lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -20,6 +24,42 @@ module.exports = function({workspace, options}) {
 		pathPrefix = `/resources/${options.namespace}`;
 	}
 
+	function updateJson(data) {
+		const mLibs = data["sap.ui5"].dependencies.libs;
```

Comment:
can't it happen that there's no dependencies/libs section yet in the manifest?

Reply:
at least if createManifest > createSapUi5 > create  > depdenencies is running / the creator is doing everything right, it should be fine.
In general the dependencies is flagged as mandatory while I only found a "have to be filled ... for ui5 apps" for the libs section.

Reply:
safety first... now within the mandatory part we always have the section not explicitly mentioned as mandatory

Reply:
As most mandatory sections are not checked at runtime, I would expect the tooling to do the same. It should not expect that the given manifest is valid w.r.t. the schema.